src/codec/h264/nalu.rs

```diff
@@ -97,6 +98,18 @@ where
             .windows(3)
             .position(|window| window == [0x00, 0x00, 0x01])
     }
+
+    pub fn into_owned(self) -> Nalu<'static, U>
+    where
+        U: Clone,
+    {
+        Nalu {
+            header: self.header.clone(),
```

Comment:
You are taking ownership of `self` here, so you don't need to call `.clone()` - moving `self.header` out will be sufficient.

This also allows you to remove the `U: Clone` constraint on this method.

Reply:
Oh, you're right. The first version was supposed to be like `to_owned()`. It's changed now.